src/Razor/src/Microsoft.CodeAnalysis.Razor.Workspaces/Protocol/DelegatedTypes.cs

```diff
@@ -39,6 +39,11 @@ internal record DelegatedValidateBreakpointRangeParams(
     [property: JsonPropertyName("projectedRange")] LspRange ProjectedRange,
     [property: JsonPropertyName("projectedKind")] RazorLanguageKind ProjectedKind) : IDelegatedParams;
 
+internal record DelegatedDataTipRangeParams(
```

Comment:
We have `DelegatedPositionParams` defined above for simple requests that just take a document, position and language. Could use that instead of defining a new type.

Reply:
Nice! I didn't notice that.